hnn_core/network_builder.py

```diff
@@ -507,7 +498,11 @@ def aggregate_data(self):
         """Aggregate somatic currents, voltages, and dipoles."""
         for cell in self._cells:
             if cell.name in ('L5Pyr', 'L2Pyr'):
-                self.dipoles[_long_name(cell.name)].add(cell.dipole)
+                nrn_dpl = self.dipoles[_long_name(cell.name)]
+                if nrn_dpl.size() > 0:
```

Comment:
```suggestion
                # dipoles are initialized as empty h.Vector() containers
                # the first cell encountered by this loop is "appended", setting the
                # length of the vector, to which subsequent cell data are added (sum)
                if nrn_dpl.size() > 0:
```

Should one come back to this and go "WTF?" ;)